docs/dashboard/team-roles-and-permissions.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Team Roles and Permissions
+description: Learn about the different roles available for team members on a Clerk Dashboard.
+---
+
+**Clerk Dashboard** offers various roles to manage team members' access. Different roles ensure that team members have the appropriate level of access according to their responsibilities.
+
+### Roles and Permissions
+
+Refer to the permissions that each role provides mapped to Dashboard resources:
+
+| Resource | Action | Admin | Member | Developer | Support |
+| - | - | - | - | - | - |
+| **Users** | Read | ✓ | ✓ | ✓ | ✓ |
+| **Users** | Manage | ✓ | | | |
+| **Users** | Impersonate | ✓ | | | ✓ |
+| **Users** | Ban | ✓ | | | ✓ |
+| **Organizations** | Read | ✓ | ✓ | ✓ | ✓ |
+| **Organizations** | Manage | ✓ | | | |
+| **Organization settings** | Read | ✓ | ✓ | ✓ | ✓ |
+| **Organization settings** | Manage | ✓ | | ✓ | |
+| **Applications** | Manage | ✓ | | | |
+| **Developer Resources** | Manage | ✓ | | ✓ | |
+| **API Keys** | Manage | ✓ | | | |
+| **Billing** | Read | ✓ | ✓ | ✓ | ✓ |
+
+## How to Change a Team Member's Role
+
+To change a team member's role, you must be an **Admin**. Follow these steps in the `<OrganizationProfile />` component:
```

Comment:
I think we shouldn't mention the SDK component here on a Dashboard walkthrough, it's confusing to guide the users to look for it.